src/assets/schemas/common/common.json

```diff
@@ -187,6 +187,27 @@
       "type": "object",
       "nullable": true,
       "sanitize": "mongo"
+    },
+    "tenantSupport": {
```

Comment:
To move tenantSupport -> tenant.json ?
Why in common.json as it's specific to the Tenant object?

Reply:
no specific reson, I just didn't realized i left it here 😇